apps/create-fuels-counter-guide/package.json

```diff
@@ -14,8 +14,8 @@
     "postbuild": "run-s fuels:build original:build"
   },
   "dependencies": {
-    "@fuels/connectors": "^0.27.1",
-    "@fuels/react": "^0.27.1",
+    "@fuels/connectors": "^0.36.1",
+    "@fuels/react": "^0.36.1",
```

Comment:
@petertonysmith94 I suppose https://github.com/FuelLabs/fuels-ts/issues/3494 wasn't blocked then or has something changed in the connectors?

Reply:
The two issues identified:

> The burner wallet connector ignores the fuelProvider passed in via the burnerWalletConfig and is always connected to testnet for some reason. This is causing it to be unusable in our UI on localhost.

Passing the `fuelProvider` straight into the `defaultConnectors` rather than into `burnerWalletConfig` resolved this. Going to create an issue + PR to make this config more intuitive in the connectors repo.

> The 'You have no funds - please bridge funds to Ignition' popup appears even when the user is not connected to Ignition, eg. localhost.

Needed to disable the `suggestBridge` in the UI config.

Reply:
It looks like it's already due for [deprecation](https://github.com/FuelLabs/fuel-connectors/blob/main/packages/burner-wallet-connector/src/types.ts#L4).